application/app.py

```diff
@@ -198,12 +199,35 @@ def images_exist(self, folder):
         return False
 
 
+    def upload(self):
+        project_name = ac.CURRENT_PROJECT_PATH.split('/')[-1].strip('/')
```

Comment:
What does this `.strip('/')` do if we've already split by `/`? (I might be forgetting some elementary Python)